File: python/tugas per9/input_output.py
nama = "buku"
harga = 10000
jumlah = 3

total = harga * jumlah

if total > 276000:
    diskon = total * 0.06   
else: 
    diskon = 0   
total_bayar = total - diskon

print("\n===== STRUK PEMBELIAN =====")
print(f"Nama Barang   : {nama}")
print(f"Harga Satuan  : {harga}")
print(f"Jumlah Beli   : {jumlah}")
print(f"Total Harga   : {total}")
print(f"Diskon        : {diskon}")
print(f"Total Bayar   : {total_bayar}")
print("===========================")

File: c++/tugas per9/input_output.cpp
#include <iostream>
using namespace std;

int main() {
    string nama = "Buku";
    float harga = 10000;
    int jumlah = 3;

    float total = harga * jumlah;
    float diskon;

    if (total > 27600) { 
        diskon = total * 0.6;
    } else {
        diskon = 0;
    }

    float total_bayar = total - diskon;

    cout << "===== STRUK PEMBELIAN =====\n";
    cout << "Nama Barang    : " << nama << endl;
    cout << "Harga Satuan   : " << harga << endl;
    cout << "Jumlah Beli    : " << jumlah << endl;
    cout << "Total Harga    : " << total << endl;
    cout << "Diskon         : " << diskon << endl;
    cout << "Total Bayar    : " << total_bayar << endl;
    cout << "===========================\n";

    return 0;
}